src/py42/clients/securitydata.py

```diff
@@ -185,6 +185,24 @@ def search_file_events(self, query):
         """
         return self._file_event_service.search(query)
 
+    def get_all_file_events(self, query, pg_token=""):
+        """Searches for all file events.
+        `REST Documentation <https://support.code42.com/Administrator/Cloud/Monitoring_and_managing/Forensic_File_Search_API>`__
+
+        Args:
+            query (:class:`py42.sdk.queries.fileevents.file_event_query.FileEventQuery`): Also
+                accepts a raw JSON str.
+            pg_token (str): A token used to indicate the starting point for additional page results. Defaults to empty string.
+                For the first page do not pass pg_token, for all consecutive pages pass the token returned from previous response.
+
+        Returns:
+            tuple (str, :class:`py42.response.Py42Response`): A tuple of token, to fetch remaining events, and response containing events.
```

Comment:
Since the `nextPgToken` is on the response, I don't see why this needs to return a tuple

Reply:
it doesn't have to, but I thought it might be a nice convenience. We don't have any other methods that do this though, so maybe you're right and should just be consistent here. In that case, we should update the docstring to clearly indicate where the get the next value from.

Reply:
Changed. I feel since this is a different use-case it seems returning the tuple is more intuitive.

Reply:
We could add a user-guide or amend a user-guide to show an example of getting file events with the tokens..

In fact, we really should do that either way.

Reply:
Added user-guide in `searches.md`